Area51/Sources/ContributionsController.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+class ContributionController: UITableViewController {
+
+    var listOfContributors = [Contributor]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: "ContributorCell", bundle: nil), forCellReuseIdentifier: "ContributorCell")
```

Comment:
Would be nice to factor this similar to https://github.com/kgellci/Area51/blob/master/Area51/Sources/FeedViewController.swift#L42